backend/keystore/keystores.go

```diff
@@ -39,7 +39,8 @@ type Keystores interface {
 	// keystores that have a secure output.
 	OutputAddress(*signing.Configuration, coin.Coin) error
 
-	// SignTransaction signs the given proposed transaction on all keystores.
+	// SignTransaction signs the given proposed transaction on all keystores. Returns
+	// `ErrSigningAborted` if the user aborts.
```

Comment:
backticks "`" are ignored by go doc. It's not recommended to use in doc comments. The common opinion is that they make the comments less readable.

Reply:
Good to know! Done.